src/collections/_documentation/development/sdk-dev/event-payloads/breadcrumbs.md

```diff
@@ -3,14 +3,16 @@ title: Breadcrumbs Interface
 sidebar_order: 6
 ---
 
-The breadcrumbs interface specifies a series of application events, or
-`breadcrumbs`, that occurred before the main event. Its canonical name is
-`breadcrumbs`.
+The Breadcrumbs Interface specifies a series of application events, or
+"breadcrumbs", that occurred before and led up to an event.
 
-This interface is an object with a single `values` key containing an ordered
-list of breadcrumb objects. The entries are ordered from oldest to newest.
-Consequently, the last entry in the array should be the last entry before the
-event occurred.
+An [event]({%- link _documentation/development/sdk-dev/event-payloads/index.md
+-%}) may contain one or more breadcrumbs in an attribute named `breadcrumbs`.
+
+This interface is an object with a single `values` attribute containing an
+ordered list of breadcrumb objects. The entries are ordered from oldest to
```

Comment:
```suggestion
ordered list of breadcrumb objects. The list will order the entries from oldest to newest. 
```

Reply:
@MimiDumpling please help me with this one.

In "The list will order the entries from oldest to newest", I understand that "The list" is the subject, "will order" the verb, "the entries" the object.

Given that a list is a data structure that contains an ordered sequence of items, "the list" itself doesn't "order"(/"sort"?) anything. The items are intrinsically in some order.

Should I interpret the sentence differently?

As similar sentence appears in `exceptions.md`, https://github.com/getsentry/sentry-docs/pull/1554/files#diff-9b11e393b13bda7b8428e2bdd3d87becR15-R16. We should align those.

Reply:
@rhcarvalho Yeah, this was my attempt at changing the sentence to active voice by giving the sentence a clear subject. If it doesn't make technical sense, we can disregard my changes here. 😃